src/shared/test/cfi-test.js

```diff
@@ -119,4 +124,46 @@ describe('sidebar/util/cfi', () => {
       );
     });
   });
+
+  describe('cfiInRange', () => {
+    [
+      // CFI before start of range
+      {
+        cfi: '/2',
+        start: '/4',
+        end: '/6',
+        expected: false,
+      },
+      // CFI at start of range
+      {
+        cfi: '/2',
+        start: '/2',
+        end: '/3',
+        expected: true,
+      },
+      // CFI in middle of range
+      {
+        cfi: '/4',
+        start: '/2',
+        end: '/6',
+        expected: true,
+      },
+      // CFI at start and end of empty range
+      {
+        cfi: '/2',
+        start: '/2',
+        end: '/2',
+        expected: false,
+      },
+      // CFI after end of range
+      {
+        cfi: '/6',
+        start: '/2',
+        end: '/4',
+        expected: false,
+      },
+    ].forEach(({ cfi, start, end, expected }) => {
+      assert.equal(cfiInRange(cfi, start, end), expected);
```

Comment:
I'm not sure if this works anyway, but should this assert be wrapped in an `it` block?

```suggestion
      it('something something something', () => {
        assert.equal(cfiInRange(cfi, start, end), expected);
      });
```

Reply:
Aggh, indeed. The assert is called with the current code, but it runs at the wrong time!